mungegithub/mungers/sig-mention-handler.go

```diff
@@ -128,3 +136,23 @@ The `+"`<group-suffix>`"+` in the method 1 has to be replaced with one of these:
 		}
 	}
 }
+
+// isStaleIssueComment removes the needs-sig comment from issues that have the proper label
```

Comment:
isStaleIssueComment just checks if a comment is stale, it doesn't remove the comment.